megalinter/descriptors/kubernetes.megalinter-descriptor.yml

```diff
@@ -70,3 +70,28 @@ linters:
               && mv ${ML_THIRD_PARTY_DIR}/kubeconform /usr/local/bin \
               && rm ${ML_THIRD_PARTY_DIR}/kubeconform-linux-amd64.tar.gz \
               && find ${ML_THIRD_PARTY_DIR} -type f -not -name 'LICENSE*' -delete -o -type d -empty -delete
+
+  # HELM LINT
+  - linter_name: helm
+    name: KUBERNETES_HELM
+    linter_repo: https://github.com/helm/helm
+    linter_url: https://helm.sh/docs/helm/helm_lint/
+    linter_version_cache: "v3.11.1"
+    files_sub_directory: ""
+    test_folder: kubernetes_helm
+    active_only_if_file_found:
+      - Chart.yml
+      - Chart.yaml
+    cli_lint_mode: project
+    cli_help_arg_name: -h
+    cli_lint_extra_args:
+      - "lint"
+    linter_banner_image_url: https://user-images.githubusercontent.com/19731161/142411871-f695e40c-bfa8-43ca-97c0-94c256749732.png
```

Comment:
@ThomasSanson this is kubeconform logo :)

Reply:
fix : add helm logoo for linter banner

I leave it to you to resolve this conversation if you wish.